tests/login.go

```diff
@@ -16,35 +16,28 @@ import (
 	"kubevirt.io/kubevirt/tests/console"
 )
 
-// LoginToCirros call LoggedInFedoraExpecter but does not return the expecter
-func LoginToCirros(vmi *v1.VirtualMachineInstance) error {
-	expecter, err := LoggedInCirrosExpecter(vmi)
-	if err == nil {
-		expecter.Close()
-	}
-	return err
-}
+// LoginToFactory represents the LogIn* functions signature
+type LoginToFactory func(*v1.VirtualMachineInstance) error
```

Comment:
It's not a factory function anymore, also since we have remove the expecter as return value is very generic, what do you think about renaming it and move it to "console" package, something like console.LoginInFunction.